app/controllers/api/v2/courses_controller.rb

```diff
@@ -180,6 +189,10 @@ def course_params
           )
       end
 
+      def subject_params
+        params.fetch(:course, {})[:subjects]
```

Comment:
Something like that 

. Permit(subjects : [])

Reply:
Discussed with Misha. If you do it the way you suggest you have to exclude all the other attributes (somewhere around 30). If you just want 1 param returned fetch seems preferable.